README.md

```diff
@@ -134,4 +137,3 @@ c:geo is distributed under the [Apache License, Version 2.0](https://www.apache.
 - Facebook: https://www.facebook.com/android.geocaching
 - Google Play: https://play.google.com/store/apps/details?id=cgeo.geocaching
 - Live status: https://status.cgeo.org/
-- Developer chat: [#cgeo on freenode.net](https://webchat.freenode.net/#cgeo)
```

Comment:
BTW, the freenode chat is mentioned in line 13 of this file as well...

Reply:
removed as well, thanks for the hint